MobileCenterPush/MobileCenterPush/MSPush.m

```diff
@@ -287,6 +287,13 @@ - (BOOL)didReceiveRemoteNotification:(NSDictionary *)userInfo fromUserNotificati
         [self.delegate push:self didReceivePushNotification:pushNotification];
       });
 #if TARGET_OS_OSX
+    } else {
+      NSUserNotification *notification = [[NSUserNotification alloc] init];
+      notification.title = title;
+      notification.informativeText = message;
+      notification.userInfo = userInfo;
+      NSUserNotificationCenter *center = [NSUserNotificationCenter defaultUserNotificationCenter];
+      [center deliverNotification:notification];
     }
 #endif
```

Comment:
Thanks for the feedback. I hope we can revisit and refactor/cleanup all of those `#if` things. 😄